test/observer/app.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/services.dart';
+
+import '../test.dart';
+
+/// An observer for the application state. 
+class AppObserver {
```

Comment:
Rename this (and files, and the other observer) to *ChannelObserver

This one would be SystemChannelsObserver

Reply:
Done, it's more descriptive. 👍